src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <b-container>
```

Comment:
I was wondering whether it might be an idea to have defaults for the fields? We know that they need to be selected but then we validate them. If there were defaults there would be no need to validate.

Reply:
Copies and number of labels could both be 1, but might be odd for `barcode` and `printer` fields to have defaults